src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseData.cs

```diff
@@ -0,0 +1,486 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Packaging
+{
+
+    // Auto-Generated by the GenerateLicenseList tool. DO NOT EDIT this manually. Use the update script at $repositoryRoot/scripts/utils/UpdateNuGetLicenseSPDXList.ps1
+    internal class NuGetLicenseData
+    {
+        public static string LicenseListVersion = "v3.1-67-geb2589b";
+
+        public static Dictionary<string, LicenseData> LicenseList = new Dictionary<string, LicenseData>()
```

Comment:
>Dictionary [](start = 22, length = 10)

nit: IReadOnlyDictionary

Reply:
And `readonly`

---
In reply to: [218980640](https://github.com/NuGet/NuGet.Client/pull/2424#discussion_r218980640) [](ancestors = 218980640)